src/test/java/org/fundacionjala/Shapes/CircleTest.java

```diff
@@ -0,0 +1,35 @@
+package org.fundacionjala.Shapes;
+
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static junit.framework.TestCase.assertEquals;
+
+/**
+ * Test for {@Link Circle}
+ */
+public class CircleTest {
+
+    private Shapes shapes;
+    public static final double DELTA = 0.5;
+
+    @Before
+    public void setUp() {
+        final int radio = 5;
+        shapes = new Circle(radio);
+    }
+
+    @Test
+    public void test_calculateArea_verifyTheCalculationOfCircleArea() {
+       final double resultArea = 78.5;
+        assertEquals(shapes.calculateArea(), resultArea, DELTA);
```

Comment:
In fact the first argument is the expected value then the second argument is the actual value